### PR TITLE
M&p setting optimization v1

### PR DESCRIPTION
**Pull request by THeijmans**
This is the first version of the setting optimization. All these settings have been tested.

### CODE REVIEW

File: resources/definitions/ultimaker3.def.json [hunk 1]
```diff
@@ -90,6 +90,7 @@
         "infill_overlap": { "value": "0" },
         "infill_pattern": { "value": "'triangles'" },
         "infill_wipe_dist": { "value": "0" },
+		"initial_layer_line_width_factor": { "value": "120" },
```
Comment (fieldOfView) [inline resources/definitions/ultimaker3.def.json]: Please replace tabs by spaces

File: resources/definitions/ultimaker3.def.json [hunk 2]
```diff
@@ -109,7 +110,9 @@
         "material_bed_temperature": { "maximum_value": "115" },
         "material_bed_temperature_layer_0": { "maximum_value": "115" },
         "material_standby_temperature": { "value": "100" },
+		"meshfix_maximum_resolution": { "value": "0.04" },
```
Comment (fieldOfView) [inline resources/definitions/ultimaker3.def.json]: Please replace tabs by spaces

**Comment by THeijmans** [comment]
Thanks Aldo, this commit should fix it. Is it a problem that "All checks have failed" ? And do I have to do something to add this commit to the pull request?

**Comment by fieldOfView** [comment]
That’s not your fault. The tests were already broken when you started this PR.

**Pull request merged**